content/db.php

```diff
@@ -0,0 +1,1330 @@
+<?php
```

Comment:
@BronsonQuick rather than copying this file, can we just link it like we do in

```PHP
<?php

if ( ! defined( 'HM_DEV' ) || ! HM_DEV ) {
	require_once dirname( __FILE__ ) . '/mu-plugins/hyperdb/db.php';	
}
```